apps/field.go

```diff
@@ -61,6 +61,8 @@ type Field struct {
 	// TODO: ModalLabel should default to Label, Name
 	ModalLabel string `json:"modal_label"`
 
+	SelectIsMulti bool `json:"multiselect,omitempty"`
```

Comment:
Since everything is now heavily documented, should we add here some documentation too?